### ISSUE TITLE
Incorrect ComboBox DropDown position with ScaleTransform

### ISSUE THREAD

**Issue by FooPenguin** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2179]
Hi,
I noticed some awkward behavior with ComboBoxes when i use ScaleTransform. When i open DropDown, label with selected value in combobox is shifted. Only combo box with classic mode enable looks correct.

```
<UserControl.LayoutTransform>
   <ScaleTransform CenterX="0" CenterY="0" ScaleX="2" ScaleY="2" />
</UserControl.LayoutTransform>
```

![image](https://user-images.githubusercontent.com/9827932/100026659-7d702c80-2deb-11eb-9f97-d4c5c1cd7b7a.png)
![image](https://user-images.githubusercontent.com/9827932/100026696-8bbe4880-2deb-11eb-910a-1700c17b9d98.png)
![image](https://user-images.githubusercontent.com/9827932/100026725-97aa0a80-2deb-11eb-97b8-364f06eee369.png)

**Comment by FooPenguin** [comment]
#1224

**Comment by FooPenguin** [comment]
I'm fix the problem and add PR #2180
